upgrade/upgrade-1.0.1.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ *                       ######
+ *                       ######
+ * ############    ####( ######  #####. ######  ############   ############
+ * #############  #####( ######  #####. ######  #############  #############
+ *        ######  #####( ######  #####. ######  #####  ######  #####  ######
+ * ###### ######  #####( ######  #####. ######  #####  #####   #####  ######
+ * ###### ######  #####( ######  #####. ######  #####          #####  ######
+ * #############  #############  #############  #############  #####  ######
+ *  ############   ############  #############   ############  #####  ######
+ *                                      ######
+ *                               #############
+ *                               ############
+ *
+ * Adyen PrestaShop plugin
+ *
+ * Copyright (c) 2019 Adyen B.V.
+ * This file is open source and available under the MIT license.
+ * See the LICENSE file for more info.
+ */
+
+if (!defined('_PS_VERSION_')) {
```

Comment:
is it necessary here? The main adyen file has this at the beginning so if it is not available we would not be able to install it at the first place, right?

Reply:
probably to avoid direct script access? I've seen this in all modules/examples: https://devdocs.prestashop.com/1.7/modules/creation/enabling-auto-update/#script-examples
but I see now index.php is missing in this folder, and in others, will add it